app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -138,6 +144,184 @@ fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
     }
 }
 
+@Composable
+fun FrontCardFace(id: String) {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(CategoryLevelDummy.valueOf(level).backgroundColor),
+        contentAlignment = Alignment.Center,
+    ) {
+        val cardImage = painterResource(id = R.drawable.bg_card_large)
+        val category = painterResource(id = R.drawable.ic_tag_love)
+
+        Image(
+            painter = cardImage, contentDescription = null, modifier = Modifier.fillMaxSize()
+        )
+        Image(
+            painter = category,
+            contentDescription = null,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(
+                    horizontal = dimensionResource(id = R.dimen.horizontal_padding),
+                    vertical = 32.dp
+                )
+        )
+    }
+}
+
+@Composable
+fun BackCardFace() {
+    Box(
+        modifier = Modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center,
+    ) {
+        val cardImage = painterResource(id = R.drawable.card_back)
+
+        Image(painter = cardImage, contentDescription = null, modifier = Modifier.fillMaxSize())
+        Column(
+            modifier = Modifier.padding(horizontal = 10.dp, vertical = 15.dp)
+        ) {
+            Topic()
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f)
+                    .background(Color.White)
+            )
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(Gray03)
+            )
+            Starter()
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(Gray03)
+            )
+            ShareBottom()
+        }
+    }
+}
+
+@Composable
+fun Topic() {
+    Column(
+        modifier = Modifier
+            .padding(24.dp)
+    ) {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Text(
+                modifier = Modifier.align(Alignment.CenterStart),
+                text = stringResource(R.string.detail_topic),
+                color = Gray05,
+            )
+            Icon(
+                modifier = Modifier
+                    .align(Alignment.CenterEnd)
+                    .size(24.dp)
+                    .padding(2.dp)
+                    .clickable {  }
+                ,
+                painter = painterResource(id = R.drawable.ic_star_empty),
+                tint = Gray05,
+                contentDescription = null
+            )
+        }
+        Spacer(
+            modifier = Modifier
+                .height(8.dp)
+                .fillMaxWidth(),
+        )
+        Text(
+            text = "천사처럼 자고 있던 내 연인, 슬며시 다가가 머리를 쓰다듬는데 나도 몰랐던 가발이 벗겨졌다. 모르는척 다시 씌워준다 vs 왜 날 속였냐고 따진다.",
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)